services/velero/metadata.yaml

```diff
@@ -14,6 +14,11 @@ overview: |-
   Velero is an open source tool for safely backing up and restoring resources in a Kubernetes cluster, performing disaster recovery, and migrating resources and persistent volumes to another Kubernetes cluster.
 
   Velero offers key data protection features, such as scheduled backups, retention schedules, and pre- or post-backup hooks for custom actions. The tool can help protect data stored in persistent volumes and makes your entire Kubernetes cluster more resilient.
+  
+  ## Attention Grabbing Title Here
```

Comment:
Ping @charIesdu @D2iQdocs Can you ptal and suggest the content here as this will be visible to user in DKP UI.